pygeo/parameterization/DVGeo.py

```diff
@@ -2788,6 +2789,7 @@ def demoDesignVars(
 
                         # Iterate counter
                         count += 1
+        self.setDesignVars(dvDictOrig)
```

Comment:
As you said in #110, the DVs are reset, so you can probably just replace this with `self.setDesignVars(dvDict)` and not have two copies of the dictionary.

Reply:
You're right, done now